pkg/flag/scan_flags.go

```diff
@@ -32,6 +32,12 @@ var (
 		Value:      []string{types.SecurityCheckVulnerability, types.SecurityCheckSecret},
 		Usage:      "comma-separated list of what security issues to detect (vuln,config,secret,license)",
 	}
+	ComplianceFlag = Flag{
```

Comment:
@knqyf263 not sure, as it trigger scanning behind the scene , so it invoke scan process

Reply:
Yes, it also affects scanning, but it is mainly for reporting, no?

Reply:
For example, `--format cyclonedx` also affects scanning since SBOM doens't need vulnerabilities, misconfigurations and secrets, but it is under report flags as it is mainly for reporting.